drafts/dynamic-components-loading.md

```diff
@@ -12,10 +12,10 @@ ignorePost: true
 Sometimes you have to deal with handling various views for different data types. In this article, we explore dynamic components loading in Angular.
 
 ### Problem
-Let's say we are fetching a list of cameras and camera equipment for our glorious online shop. Each item is either a camera or camera equipment. You know, like tripods, lenses, bags, and whatever fancy filters come to your mind. All those items share some common attributes like price, name, some overselling description, and an id. However, when it comes to the technical characteristics, they differ. For cameras, we want to show the list of tech attributes in a table. And for the equipment, we ought to use single values, references, or whatever fancy displaying alternative comes to your mind. The point is, that we need to employ different views depending on the item type.
+Let's say we are fetching a list of cameras and camera equipment for our glorious online shop. Each list item is either a camera or camera equipment. You know, like tripods, lenses, bags, and whatever fancy filters come to your mind. All those items share some common attributes like price, name, an overselling description, and an id. However, when it comes to the technical characteristics, they differ. Therefore, we have different requirements when it comes to displaying them. For cameras, we want to show the list of tech specs in a table. And for the equipment, we ought to use single values, references, or whatever fancy displaying alternative comes to your mind. The point is, that we need to employ different views depending on the item's tech attribute type.
```

Comment:
```suggestion
Let's say we are fetching a list of cameras and camera equipment for our glorious online shop. Each list item is either a camera or camera equipment. You know, like tripods, lenses, bags, and whatever fancy filters come to your mind. All those items share some common attributes like price, name, an overselling description, and an id. However, when it comes to the technical characteristics, they differ. Therefore, we have different requirements when it comes to displaying them. For cameras, we want to show the list of tech specs in a table. And for the equipment, we ought to use single values, references, or whatever fancy displaying alternative comes to your mind. The point is, that we need to employ different views depending on the items tech attribute type.
```
I think there is no apostrophe here

Reply:
hmm... i would say it does. "the tech attribute type of the item" -> "the item's tech attribute type"

Reply:
Ah ok, sounds reasonable :)